test/petab/conftest.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import benchmark_models_petab as models
+import petab
+import pytest
+
+import pypesto.petab
+
+
+@pytest.fixture(scope="session")
+def loaded_models():
+    """Fixture to load all models used in Tests to be only loaded once."""
+    model_names = [
+        "Zheng_PNAS2012",
+        "Brannmark_JBC2010",
+        "Boehm_JProteomeRes2014",
+    ]
```

Comment:
Hm... So whenever a test uses any of these models, all of them have to be imported? I am not convinced that this is an improvement. Running only a single test is not that uncommon.

When called twice on the same model (from within the same directory), the PetabImporter should already reuse any already imported model. Is this not happening?

Moving  the PEtab import of models that are used in several tests to (individual) session-scoped fixtures makes sense though. (Ideally having both, the petab.Problem and pypesto.Problem in that fixture).

Reply:
right, did not think about only individual tests. could just write a session wide fixture for models that are used and use that fixture then.